### PR TITLE
Batch vc send error test

### PR DESCRIPTION
**Pull request by BillyWooo**
try to catch the panic/error from batch vc task.

### CODE REVIEW

File: tee-worker/enclave-runtime/src/initialization/mod.rs [hunk 1]
```diff
@@ -295,7 +295,17 @@ fn run_vc_issuance() -> Result<(), Error> {
 	);
 	let extrinsic_factory = get_extrinsic_factory_from_integritee_solo_or_parachain()?;
 	let node_metadata_repo = get_node_metadata_repository_from_integritee_solo_or_parachain()?;
-	run_vc_handler_runner(Arc::new(stf_task_context), extrinsic_factory, node_metadata_repo);
+
+	let vc_handler = std::thread::spawn(move || {
```
Comment (silva-fj) [inline tee-worker/enclave-runtime/src/initialization/mod.rs:299]: why do we need to spawn a new thread? `run_vc_issuance` is called within an already spawned thread

**Review by felixfaisal — APPROVED** [review]
Thanks @BillyWooo , I think for the temporary need to figure out the `SendError` we need this, However for a long term, it is redundant to have these two threads and we need to do this more ergonomically.

**Comment by BillyWooo** [comment]
> Thanks @BillyWooo , I think for the temporary need to figure out the `SendError` we need this, However for a long term, it is redundant to have these two threads and we need to do this more ergonomically.

Yes. Once we detect/fix it, we need to remove this wrapper layer.

File: tee-worker/enclave-runtime/src/initialization/mod.rs (hunk 1 above)
Comment (BillyWooo) [reply tee-worker/enclave-runtime/src/initialization/mod.rs:299]: Because it's hard to reproduce. This is only for test purpose. Will remove it later.

**Pull request merged**